<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitedChangerMainWindow</class>
 <widget class="QDialog" name="UnitedChangerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
</string>
  </property>
  <widget class="QComboBox" name="Value1ComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
border: 1px solid black;
border-radius: 4px;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Value2ComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
border: 1px solid black;
border-radius: 4px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogotypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid black;
border-radius: 20px;
font-size: 24px;
</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>-10</y>
     <width>16</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>881</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>20</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>621</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>400</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>199</width>
      <height>219</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelValue1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 20px;
font-size: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelValue2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 20px;
font-size: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Value1Flag">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid whitesmoke;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Value2Flag">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>160</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid whitesmoke;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEquality">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 20px;
font-size: 60px;
background-color: white;</string>
   </property>
   <property name="text">
    <string>⇄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>330</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
border-radius: 10px;
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Liked Values&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LikedInCsvButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; background-color: white; border: 1px solid black; font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Convert Liked in CSV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilenameLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 10px;
font-size: 20px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Filename...</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DoubleSpinConverterBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
border: 1px solid black;
font-size: 16px;</string>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
  </widget>
  <widget class="QLabel" name="ConvertationResult">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 20px;
font-size: 20px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Converter1ComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
border: 1px solid black;
border-radius: 4px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConvertButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
border: 1px solid black;
font-size: 40px;</string>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Converter2ComboBox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
border: 1px solid black;
border-radius: 4px;
</string>
   </property>
  </widget>
  <zorder>Value1ComboBox</zorder>
  <zorder>Value2ComboBox</zorder>
  <zorder>LogotypeLabel</zorder>
  <zorder>settingsButton</zorder>
  <zorder>line1</zorder>
  <zorder>line2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>scrollArea</zorder>
  <zorder>labelValue1</zorder>
  <zorder>labelValue2</zorder>
  <zorder>line3</zorder>
  <zorder>Value1Flag</zorder>
  <zorder>Value2Flag</zorder>
  <zorder>buttonEquality</zorder>
  <zorder>label</zorder>
  <zorder>LikedInCsvButton</zorder>
  <zorder>FilenameLine</zorder>
  <zorder>DoubleSpinConverterBox</zorder>
  <zorder>ConvertationResult</zorder>
  <zorder>Converter1ComboBox</zorder>
  <zorder>ConvertButton</zorder>
  <zorder>Converter2ComboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
